fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -651,4 +668,172 @@ public static long analyzeLongProp(Map<String, String> properties, String propKe
         }
         return val;
     }
+
+    public static List<UniqueConstraint> analyzeUniqueConstraint(
+            Map<String, String> properties, OlapTable table) throws AnalysisException {
+        List<UniqueConstraint> uniqueConstraints = Lists.newArrayList();
+        if (properties != null && properties.containsKey(PROPERTIES_UNIQUE_CONSTRAINT)) {
```

Comment:
can add unique constraints for `UNIQUE/Primary key` tables?

Reply:
yes. we can add unique constraints for unique/primary key tables. because a table can have multi unique constraints, but only one primary key.